Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM openmicroscopy/ome-files-cpp-u1604
```

Comment:
As a general benchmarking comment, is it appropriate to use virtualisation here?  If we want to compare the bare metal Windows performance, would it not be better to run on the bare metal as well?